src/sentry/incidents/action_handlers.py

```diff
@@ -33,6 +33,14 @@ def resolve(self, metric_value):
         pass
 
 
+class DefaultActionHandler(ActionHandler):
```

Comment:
Actually, this should implement a `send_alert` abstract method since we're calling it from fire/resolve

Reply:
I wanted to think of a slightly better way to do this because right now the only difference between `send_alert` in all of the classes that extend `DefaultActionHandler` is where they import `send_incident_alert_notification` from. Added the abstract method for now :-)